sklearn/ensemble/tests/test_bagging.py

```diff
@@ -723,3 +723,26 @@ def test_max_samples_consistency():
                                 max_features=0.5, random_state=1)
     bagging.fit(X, y)
     assert_equal(bagging._max_samples, max_samples)
+
+
+def test_multi_output_regressor():
+    # Check singleton ensembles.
+    rng = check_random_state(0)
+    X_train, X_test, y_train, y_test = train_test_split(boston.data,
+                                                        boston.target,
+                                                        random_state=rng)
+
+    reg1 = BaggingRegressor(base_estimator=Ridge(), n_estimators=1,
```

Comment:
Ya you're absolutely right. `n_estimaors=1` makes no sense within a test called "*bagging*" :). Fixed.